base_export_async/tests/test_base_export_async.py

```diff
@@ -64,22 +64,26 @@ def test_export_csv(self):
         params = json.loads(data_csv.get("data"))
         mails = self.env["mail.mail"].search([])
         attachments = self.env["ir.attachment"].search([])
-        self.delay_export_obj.export(params)
+        self.delay_export_obj.export(
+            params, self.env.ref("base_export_async.delay_export_mail_template"), True
```

Comment:
`self.env.ref("base_export_async.delay_export_mail_template")` is default value for ` mail_template` param, you should write a new test for `is_export_file_attached_to_email=True`